installer/mac/copy_x64_binary.py

```diff
@@ -22,10 +22,19 @@ def copy_x64(x64_src_path, x64_dest_path):
     if (os.path.exists(x64_dest_path)):
         shutil.rmtree(x64_dest_path)
 
+    if brave_channel == 'release':
+        channel = ''
+    else:
+        channel = ' %s' % brave_channel.capitalize()
+
     shutil.copytree(x64_src_path, x64_dest_path, symlinks=True,
         ignore=shutil.ignore_patterns('Sparkle.framework'))
     # remove conflicting files
     os.remove(os.path.join(x64_dest_path, 'Contents', 'Info.plist'))
+    os.remove(os.path.join(x64_dest_path, 'Contents', 'Frameworks',
```

Comment:
maybe can use `ignore_patterns` as above and skip the remove

Reply:
as discussed in DM this should not be needed if you are using the `.app` from the root of the x64 build

Reply:
also just fyi for future reference it would have been much better to pass in the path to resources from gn than to calculate it here.